### PR TITLE
Use acceptance testing transport

### PR DESCRIPTION
**Pull request by timbussmann**
according to @SzymonPobiega, there is interest in using the new acceptance testing transport instead of other transport for the acceptance tests.

This PR uses the acceptance testing transport instead of SQL and MSMQ transport to ensure that both timeout and delayed delivery persistence options are tested.

ping @Particular/sqlpersistence-maintainers

### CODE REVIEW

File: src/MsSqlAcceptanceTests/When_creating_new_transaction_scope_in_the_pipeline.cs [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-using NServiceBus.AcceptanceTests;
-using NServiceBus.AcceptanceTests.EndpointTemplates;
-using NServiceBus.Persistence.Sql;
-using NServiceBus.Pipeline;
-using NUnit.Framework;
-
-public class When_creating_new_transaction_scope_in_the_pipeline : NServiceBusAcceptanceTest
-{
-    [Test]
-    public async Task Should_fail_when_creating_synchronized_storage_session()
```
Comment (timbussmann) [inline src/MsSqlAcceptanceTests/When_creating_new_transaction_scope_in_the_pipeline.cs]: this test scenario wouldn't be supported by the acceptance testing transport

**Comment by bording** [comment]
I believe this should be against the release branch, not develop.

**Comment by bording** [comment]
@timbussmann There are still packagereferences to NServiceBus.SqlServer and NServiceBus.Transport.Msmq that can be removed.

Also, PostgreSqlAcceptanceTests hasn't been converted yet.

**Comment by timbussmann** [comment]
rebased the PR and moved all acceptance tests to the acceptance testing transport.

**Comment by SzymonPobiega** [comment]
Let me know when it is ready for review or when you have any problems

**Comment by timbussmann** [comment]
@SzymonPobiega this should be good for review now. There seem to be some issues on the build server though. I'll try to queue a new build before looking deeper.

**Pull request merged**